### DISCUSSION TITLE
About translate blog post

### DISCUSSION THREAD

**Discussion by emewjin** [facebook/stylex#200] [General]
Hello, @nmn @necolas !
I'm Yujin Lim, a frontend developer from [Korean FE Article Team](https://kofearticle.substack.com/about).

I would like to ask for your permission to translate the blog post ["Introducing StyleX"](https://stylexjs.com/blog/introducing-stylex) into Korean and publish it on [my blog](https://emewjin.github.io/) and [Korean](https://kofearticle.substack.com/)[ FE Article mailing substack](https://kofearticle.substack.com/). Of course, a link to the original article will be provided.

Korean FE Article is an initiative that selects quality frontend articles every week, translates them into Korean, and shares them. It is not used for commercial purposes, and proper source attribution is ensured.

I hope to make this valuable content accessible to Korean developers. Please consider this request, and I look forward to your reply :)

p.s: I couldn't find your email addresses, so I leave in the discussion. I hope your understand.

**Comment by nmn** [comment]
Yes. As long as the original source of the article is clear, you have our permission to re-publish a Korean translation.
